src/utils/utils.jl

```diff
@@ -464,8 +496,10 @@ function transform_array_for_hdf(data::Vector{POLYNOMIAL})
     return t_lin_cost
 end
 
-function transform_array_for_hdf(data::SortedDict{Dates.DateTime, Vector{PWL}})
-    quad_cost = hcat(values(data)...)
+function transform_array_for_hdf(
+    data::SortedDict{Dates.DateTime, Vector{PiecewiseLinearPointData}},
+)
+    quad_cost = hcat([get_points.(v) for v in values(data)]...)
```

Comment:
Why is this called `quad_cost` for a PWL data? I see that the name is prior to this PR but it seems incorrect.

Reply:
It does seem incorrect. I'll change it